src/Resources/fixtures/.gitignore

```diff
@@ -0,0 +1,4 @@
+*
```

Comment:
Seems redundant?

Reply:
This allows every developer to keep his own fixture without worrying about putting them in the repository.

Reply:
But is correlated anyhow with this task? I think we should rather not do changes on the occasion. It's not a very good practice with internal projects and especially with OS ones. It's really hard to understand why the change has been implemented as it just "happened" along the way